src/RecyclableMemoryStreamManager.cs

```diff
@@ -632,6 +639,12 @@ internal void ReturnBlock(byte[] block, Guid id, string tag)
             }
         }
 
+        private void ZeroOutMemoryIfEnabled(Span<byte> buffer) {
+            if (this.ZeroOutBuffer) {
+                buffer.Clear();
+            }
+        }
```

Comment:
We don't want to construct a span when we don't have to.

```suggestion
        [MethodImpl(MethodImplOptions.AggressiveInlining)]
        private void ZeroOutMemoryIfEnabled(byte[] buffer) {
            if (this.ZeroOutBuffer) {
                Array.Clear(buffer);
            }
        }
```

Reply:
Thanks. Chankged the code